openapi_tester/clients.py

```diff
@@ -0,0 +1,26 @@
+"""Subclass of ``APIClient`` using ``SchemaTester`` to validate responses."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from rest_framework.test import APIClient
+
+if TYPE_CHECKING:
+    from rest_framework.response import Response
+
+    from .schema_tester import SchemaTester
+
+
+class OpenAPIClient(APIClient):
+    """``APIClient`` validating responses against OpenAPI schema."""
+
+    def __init__(self, *args, schema_tester: SchemaTester, **kwargs) -> None:
```

Comment:
@sondrelg I was thinking about making `schema_tester` optional and adding some method e.g. `_schema_tester_factory` to call when it's `None`, just to make the API more developer-friendly. Please let me know what you think about it.

Reply:
Since implementations of the client will only need a single extra line of code (most likely copy pasted from the docs), I'm not sure it's worth the extra effort, but if you want to add it, I'm not opposed to it 🙂